ci_support/run_docker_build.sh

```diff
@@ -30,76 +32,57 @@ cat << EOF | docker run -i \
                         pelson/obvious-ci:latest_x64 \
                         bash || exit $?
 
-# FIXME: We needed to do this to test with QT. This is not ideal - instead, qt should be built properly.
-yum install -y libXext libXrender libSM tk libX11-devel
-
+export BINSTAR_TOKEN=${BINSTAR_TOKEN}
```

Comment:
Hadn't noticed this - we weren't creating new binaries for mpl as this wasn't set properly.